code/modules/admin/verbs/pray.dm

```diff
@@ -35,6 +35,9 @@
 	for(var/client/X in GLOB.admins)
 		if(check_rights(R_EVENT,0,X.mob))
 			to_chat(X, msg)
+			if(X.prefs.sound & SOUND_PRAYERNOTIFY)
+				X << 'sound/items/PDA/ambicha4-short.ogg'

```

Comment:
Use SEND_SOUND instead

Reply:
which part do I plug that into?

Reply:
the part thats sending the sound, line 39

Reply:
so something like this? SEND_SOUND(X << 'sound/items/PDA/ambicha4-short.ogg')

Reply:
you should search the code for examples
https://github.com/ParadiseSS13/Paradise/blob/f8662f0833dd5aeaecb865be5ac9dcf200a4fe7f/code/controllers/subsystem/afk.dm#L65

Reply:
yes like that